serverless/develop/serverless-kafka-developer.adoc

```diff
@@ -6,6 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Knative Kafka provides integration options for you to use supported versions of the Apache Kafka message streaming platform with {ServerlessProductName}. Kafka can be used as a replacement for traditional message brokers like the Knative broker. It also provides options for event source, channel, and event sink capabilities.
```

Comment:
the knative broker is not a traditional message broker.

The Knative Broker (with the Trigger) is a set of APIs for "event orchestration". The upstream calls the broker an "event hub".

I think we can just skip the 
> Kafka can be used as a replacement for traditional message brokers like the Knative broker.

sentence.